client/src/tokens/colors.js

```diff
@@ -19,179 +21,225 @@ const colors = {
       textUtility: 'w-text-grey-600',
+      cssVariable: '--w-color-grey-600',
       usage: 'Body copy, user content',
       contrastText: 'white',
     },
     400: {
       hex: '#5C5C5C',
+      hsl: 'hsl(0 0% 36%)',
       bgUtility: 'w-bg-grey-400',
       textUtility: 'w-text-grey-400',
+      cssVariable: '--w-color-grey-400',
       usage: 'Help text, placeholders, meta text, neutral state indicators',
       contrastText: 'white',
     },
     200: {
       hex: '#929292',
+      hsl: 'hsl(0 0% 57%)',
       bgUtility: 'w-bg-grey-200',
       textUtility: 'w-text-grey-200',
+      cssVariable: '--w-color-grey-200',
       usage: 'Dividers, button borders',
       contrastText: 'primary',
     },
     100: {
       hex: '#E0E0E0',
+      hsl: 'hsl(0 0% 88%)',
       bgUtility: 'w-bg-grey-100',
       textUtility: 'w-text-grey-100',
+      cssVariable: '--w-color-grey-100',
       usage: 'Dividers, field borders, panel borders',
       contrastText: 'primary',
     },
     50: {
       hex: '#F6F6F8',
+      hsl: 'hsl(240 12% 97%)',
       bgUtility: 'w-bg-grey-50',
       textUtility: 'w-text-grey-50',
+      cssVariable: '--w-color-grey-50',
       usage: 'Background for panels, row highlights',
       contrastText: 'primary',
     },
   },
   white: {
     DEFAULT: {
       hex: '#FFFFFF',
+      hsl: 'hsl(0 0% 100%)',
       bgUtility: 'w-bg-white',
       textUtility: 'w-text-white',
+      cssVariable: '--w-color-white',
       usage: 'Page backgrounds, Panels, Button text',
       contrastText: 'primary',
     },
   },
-  teal: {
+  primary: {
+    DEFAULT: {
+      hex: '#2E1F5E',
+      hsl: 'hsl(254 50% 25%)',
+      bgUtility: 'w-bg-primary',
+      textUtility: 'w-text-primary',
+      cssVariable: '--w-color-primary',
+      usage: 'Wagtail branding, Panels, Headings, Buttons, Labels',
+      contrastText: 'white',
+    },
+    200: {
+      hex: '#261A4E',
+      hsl: 'hsl(254 50% 20%)',
+      bgUtility: 'w-bg-primary-200',
+      textUtility: 'w-text-primary-200',
+      cssVariable: '--w-color-primary-200',
+      usage:
+        'Accent for elements used in conjunction with primary colour in sidebar',
+      contrastText: 'white',
+    },
+  },
+  secondary: {
     600: {
       hex: '#004345',
-      bgUtility: 'w-bg-teal-600',
-      textUtility: 'w-text-teal-600',
+      hsl: 'hsl(182 100% 14%)',
+      bgUtility: 'w-bg-secondary-600',
+      textUtility: 'w-text-secondary-600',
+      cssVariable: '--w-color-secondary-600',
       usage: 'Hover states for two-tone buttons',
       contrastText: 'white',
     },
     400: {
       hex: '#005B5E',
-      bgUtility: 'w-bg-teal-400',
-      textUtility: 'w-text-teal-400',
+      hsl: 'hsl(182 100% 18%)',
+      bgUtility: 'w-bg-secondary-400',
+      textUtility: 'w-text-secondary-400',
+      cssVariable: '--w-color-secondary-400',
       usage: 'Two-tone buttons, hover states',
       contrastText: 'white',
     },
-    200: {
+    DEFAULT: {
```

Comment:
Is it a little weird that DEFAULT corresponds to 200 for the secondary color but 100 for the primary color?

Reply:
Personally I don’t find it too weird, just arbitrary. We could keep `secondary-200` rather than have a default secondary color? I only changed this for the sake of matching how our previous teal colors were defined, with `$color-teal` corresponding to this "200" shade.